artifactory/commands/config.go

```diff
@@ -113,12 +113,6 @@ func (cc *ConfigCommand) Config() error {
 		return err
 	}
 
-	if cc.encPassword {
```

Comment:
Moving the encryption from here means password provided in interactive=false will not be encrypted

Reply:
You're right. Thanks.